tests/decode/test5_corner_cases/src/main.c

```diff
@@ -1376,6 +1378,91 @@ void test_prelude(void)
 	zassert_equal(ZCBOR_SUCCESS, cbor_decode_Prelude(prelude_payload1, sizeof(prelude_payload1), &result, &num_decode), NULL);
 }
 
+void test_cbor_bstr(void)
+{
+	uint8_t cbor_bstr_payload1[] = {
+#ifdef TEST_INDEFINITE_LENGTH_ARRAYS
+		0x58, 32,
+#else
+		0x58, 31,
+#endif
+			LIST(3),
+				0x46, 0x65, 'H', 'e', 'l', 'l', 'o',
+				0x49, 0xFB, 0x40, 0x9, 0x21, 0xca, 0xc0, 0x83, 0x12, 0x6f /* 3.1415 */,
+				0x4C, 0xC2, 0x4A, 0x42, 2, 3, 4, 5, 6, 7, 8, 9, 10 /* 0x4202030405060708090A */,
+			END
+	};
+
+	uint8_t cbor_bstr_payload2_inv[] = {
+#ifdef TEST_INDEFINITE_LENGTH_ARRAYS
+		0x58, 32,
+#else
+		0x58, 31,
+#endif
```

Comment:
Are the details of these lines important? Or would it be better to pull it out to a `define` to improve readability?

Reply:
Good idea, made a define